perms/go.mod

```diff
@@ -0,0 +1,190 @@
+module github.com/lightninglabs/lightning-terminal/perms
```

Comment:
Another option would be to just move this into the `litrpc` package, to avoid needing yet another submodule. But not sure if it fits there 100% content/topic wise. So just a thought.

Reply:
Okay, chatted with @ellemouton offline to gauge opinions on this. And the decision was to keep `perms` as its separate module, so you can ignore my comment above :+1: